TestsFoundation/TestsFoundation/Element/Element.swift

```
@@ -127,32 +134,29 @@ public struct XCUIElementWrapper: Element {
         return element.exists
     }
     public var frame: CGRect {
-        guard exists else { return .zero }
         return element.frame
```

won't this crash if the element doesn't exist? the exist guards were made so the test fails instead of crashing.

---

I'm not sure. I don't like the silently returning a valid rect though. How would you feel about `XCTAssertTrue(exists)`?

---

frame could use a waitToExist() like label & value.